README.md

```diff
@@ -232,6 +233,58 @@ This parameter can be configured to activate the strategy only when the number o
 are above the configured value. This could be helpful in large clusters where a few nodes could go
 under utilized frequently or for a short period of time. By default, `numberOfNodes` is set to zero.
 
+### HighNodeUtilization
+
+This strategy finds nodes that are under utilized and evicts pods in the hope that these pods will be scheduled compactly into fewer nodes. This strategy can be used in tandem with the 
+scheduler strategy `MostRequestedPriority`. The parameters of this strategy are configured under `nodeResourceUtilizationThresholds`.
```

Comment:
Ideally this strategy *must* be used with `MostRequestedPriority` (actually called `NodeResourcesMostAllocated` in the [plugin framework](https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins-1)). Using `HighNodeUtilization` without the matching scheduler plugin will cause the evicted pods to be rescheduled back onto the lowest-utilized nodes